lib/howitzer/meta/base.rb

```diff
@@ -0,0 +1,28 @@
+module Howitzer
+  module Meta
+    # Base class for meta elements, contains utility methods
+    class Base
+      # Highlights element with red border on the page
+      def highlight
+        if xpath.blank?
+          Howitzer::Log.info("Element #{@name} not found on the page")
+          return
+        end
+        context.execute_script("document.evaluate('#{escape(xpath)}', document, null, " \
+                          'XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.style.border = "thick solid red"')
```

Comment:
Does it work in all browsers?

Reply:
I've checked it in Edge, Chrome and Firefox. Works fine with them.